deliverables/README.md

```diff
@@ -1,68 +1,140 @@
+In order to deliver homework consisting of questions and answers, just edit this file and commit the changes.  Once your answers are on Github, specifically in this file, you have delivered your homework.
 
-Example:  
+To answer questions, please just put the answer on the next line after the question.  Put a blank line after the answer to separate it visually from the next question.
+
+Example:
 
 **What is your name?**
-Sir Student of Ruby
+Christian Dean
 
 **What is your quest?**
-I seek the Rails  
+I want to learn more about code and how to utilize tools to make my business succeed.
 
 **What is your favorite color?**
-~~Green~~ edit:  No, blue.  Blue!  
+Green is my favorite color!
 
-For projects on Code Academy, or other external exercises, please just paste a link into your deliverables file where we can see the result.  
+For projects on Code Academy, or other external exercises, please just paste a link into your deliverables file where we can see the result.
 
 ### Section 1: The Front End
-#### Required - Answer these questions in the deliverables file, and complete the following. 
-What is the role of HTML in a web page?  
-What is the role of CSS in a web page?  
-What is the role of Javascript in a web page?  
+#### Required - Answer these questions in the deliverables file, and complete the following.
+
+What is the role of HTML in a web page?
+HTML is the language, or code, used to edit and position the text, images, frames and other web page elements. Its purely for content.
+What is the role of CSS in a web page?
+CSS is an enhanced form of HTML, and makes it easier to edit the design of a web page.
+What is the role of Javascript in a web page?
+Java script primarily to make a website more active or visual, it can not go without html / css and works with it but is seperate from html and css.
 
 ---
 
 ### Section 2: HTML and CSS 101
 
 #### meta
-Why do we separate HTML and CSS?  
+Why do we separate HTML and CSS?
 
+That way it is easy to distinguish the content we are editing rather than the design of the code.
 #### HTML
-What are classes and IDs (and how are they different)?  
-What are elements?  
-What are tags?  
-What are attributes?  
-What are forms?  
-What is a div?  
-
-What are selectors?  
-What are properties?  
-What are values?  
-How do CSS styles for a particular element get inherited? ie. how does an element get its "default" styles?  
-What are two CSS attributes you can change to push an element around on the page?  
-What are the three different ways to include CSS in your project or use CSS to style a particular element?  
+What are classes and IDs (and how are they different)?
+ID's are unique throughout the code and allow you to label and edit one aspect while Classes are not unique and allow you to access and edit more than one portion of code at a time.
```

Comment:
Just one correction.  You keep mentioning "edit", but really what's happening is that the page is being "styled".  "Edit" refers to changing the contents of a file or data structure.  What CSS "changes" is the presentation of content in the browser.